### ISSUE TITLE
🪲 Bug Report - Incorrect resource type and count in slide12

### ISSUE THREAD

**Issue by tksh164** [Azure/Azure-Proactive-Resiliency-Library-v2#285]
## Describe the bug

An incorrect resource type and count inserted into the slide 12 that generated by `3_wara_reports_generator.ps1`.

## To Reproduce

Steps to reproduce the behaviour:

1. Run the `1_wara_collector.ps1` against a subscription that has less resource types and it has Network Watcher resrouces.
2. Run the `2_wara_data_analyzer.ps1`.
3. Run the `3_wara_reports_generator.ps1`.

## Expected behaviour

The correct resource type and its count will be inserted.

## Screenshots 📷

![image](https://github.com/user-attachments/assets/1b017516-7fcf-451e-8f34-207bcb9c2103)

## Additional context

This issue came from the Network Watcher's recommendation (4e133bd0-8762-bc40-a95b-b29142427d73). The recommendation has no resource ID (It has `n/a` as a resource ID).

![image](https://github.com/user-attachments/assets/20a0891a-0343-4903-b3aa-0d7707b18d8f)

The inserted content (resource type and count) is generated by unique resource ID, so the Network Watcher's resource type to be `/` and aggregated count is 1 even if there are multiple Network Watcher resources.

https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/blob/6b9d9938a30c65a336c6f635c9770ea49ac38547/tools/3_wara_reports_generator.ps1#L203-L221

**Comment by oZakari** [comment]
@Azure/aprl-wara-tools please review.

**Comment by kpoineal** [comment]
@tksh164 Is this still an issue? I believe we fixed the n/a issue in a previous release.

**Comment by tksh164** [comment]
@kpoineal Yes, this issue still exists in the released **3_wara_reports_generator.ps1**. I believe that the fix by the previous release is in **1_wara_collector.ps1**. Created [a PR for this issue](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/pull/397). Please merge the PR if it is acceptable.

The fix by the previous release is in **1_wara_collector.ps1**:
https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/blob/691f4e2aecbea80191e6d25bd64d1f1f16ca6d05/tools/1_wara_collector.ps1#L1298-L1317

The point for need fixing for this issue:
https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/blob/691f4e2aecbea80191e6d25bd64d1f1f16ca6d05/tools/3_wara_reports_generator.ps1#L263-L281

```powershell
$obj = @{
    'ID'             = $ID;  # $ID is 'n/a' for the recommendation 4e133bd0-8762-bc40-a95b-b29142427d73.
    'Subscription'   = $ID.split('/')[2];
    'Resource Group' = $ID.split('/')[4];
    'Resource Type'  = ($ID.split('/')[6] + '/' + $ID.split('/')[7])  # '' + '/' + '' if $ID is 'n/a'.
}
```

You can test by add the following to the collector's JSOIN file.

```json
{
    "validationAction": "APRL - Queries",
    "recommendationId": "4e133bd0-8762-bc40-a95b-b29142427d73",
    "name": "n/a",
    "id": "n/a",
    "type": "n/a",
    "location": "n/a",
    "subscriptionId": "n/a",
    "resourceGroup": "n/a",
    "param1": "LocationMisingNetworkWatcher:southeastasia",
    "param2": "",
    "param3": "",
    "param4": "",
    "param5": "",
    "checkName": "",
    "selector": "APRL"
}
```

**Comment by tksh164** [comment]
Fixed by #397
